### ISSUE TITLE
Doesn't work on Void

### ISSUE THREAD

**Issue by kkga** [JojiiOfficial/rsv#5]
Getting the following output on Void Linux:
```
> sudo rsv list
Can't find runsvdir! make sure you have a running 'runsvdir' process!
```

**Comment by JojiiOfficial** [comment]
Just tested it in a VM and it worked for me with a fresh installation. Try to compile the latest commit. I assume the PR fixed this issue

**Comment by privacy313** [comment]
I am having the same trouble using Artix linux. Installed using cargo, which I assume is up to date?

**Comment by JojiiOfficial** [comment]
Can you compile the git version?

The `cargo install` version is equal to the git tag (so not up to date with the master branch)

**Comment by JojiiOfficial** [comment]
This issue should've been fixed with v1.3.2. If the error still persists, reopen the issue.

**Comment by Zakyramadhan1** [comment]
i got the same issue with user service
